src/Cache/Engine/RedisEngine.php

```diff
@@ -135,6 +148,65 @@ protected function _connect(): bool
         return $return;
     }
 
+    /**
+     * Connects to a Redis server using a new connection.
+     *
+     * @param string $server Server to connect to.
+     * @throws \RedisException
+     * @return bool True if Redis server was connected
+     */
+    protected function _connectTransient($server): bool
+    {
+        if (empty($this->_config['context'])) {
```

Comment:
```suggestion
    {
        $context = [];
        $map = [
            'ssl_ca' => 'cafile',
            'ssl_key' => 'local_pk',
            'ssl_cert' => 'local_cert',
        ];
        foreach ($map as $from => $to) {
            if (!empty($config[$from)) {
                $context[$to] = $config[$from];
            }
        }
        if (empty($context)) {
```

This should do what I had proposed above.